tests/foreman/ui/test_discoveryrule.py

```diff
@@ -727,10 +836,17 @@ def test_negative_delete_rule_with_non_admin_user(self):
         @Assert: User should validation error and rule should not be deleted
         successfully.
 
-        @caseautomation: notautomated
-
         @CaseLevel: Integration
         """
+        with Session(
+            self.browser, self.username, self.user_password
+        ) as session:
+            session.nav.go_to_discovery_rules()
+            element = self.discoveryrules.find_element(
+                locators['discoveryrules.disabled_name'] % self.rule_name)
+            self.assertIsNotNone(element)
+            with self.assertRaises(UIError):
```

Comment:
`UIError` is a pretty global class, can we check for some exact error message too